keyboards/keyap/keymaps/via/rules.mk

```
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

```suggestion
ENCODER_ENABLE = yes
ENCODER_MAP_ENABLE = yes
```
This is probably why the build is failing.